tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh

```diff
@@ -23,6 +23,7 @@ readonly TEST_DRIVER_REPO_URL="https://github.com/grpc/grpc.git"
 readonly TEST_DRIVER_BRANCH="${TEST_DRIVER_BRANCH:-master}"
 readonly TEST_DRIVER_PATH="tools/run_tests/xds_k8s_test_driver"
 readonly TEST_DRIVER_PROTOS_PATH="src/proto/grpc/testing"
+readonly TEST_DRIVER_REPO_DIR_USE_EXISTING=1
```

Comment:
1. Does it work without setting `TEST_DRIVER_REPO_DIR`?
2. It's better to define it in repo-specific script, https://github.com/grpc/grpc/blob/feff79abc76a7aa821ffa305dfd5ef06e2156d8a/tools/internal_ci/linux/grpc_xds_k8s.sh#L28

Reply:
No, it errors out because the directory is already there.

Reply:
Misread the question.
> 1) Does it work without setting TEST_DRIVER_REPO_DIR?
Yes, because `TEST_DRIVER_REPO_NAME` is the same as `GITHUB_REPOSITORY_NAME`

> It's better to define TEST_DRIVER_REPO_DIR_USE_EXISTING in repo-specific script next to,
Done